### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by praveen4g0**
My first commit

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
 # Voting app for Tekton Pipeline examples
+
```
Comment (praveen4g0) [inline README.md:2]: Why Do you need extra spacing?

Comment (VeereshAradhya) [inline README.md:2]: new line is not required.. please remove it

**Review by VeereshAradhya — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (savitaashture) [inline README.md:2]: Remove extra line

Comment (VeereshAradhya) [inline README.md:2]: remote plus sign :D

**Pull request closed without merging**